docs/jira-integrations.md

```diff
@@ -0,0 +1,267 @@
+# JIRA Integrations
+
+---
+
+A test suite can be integrated with JIRA, in order to get notified if a test fails or warns. In order to use JIRA Integration, you first need to created a JIRA Authentication, than use that authentication in order to create a JIRA Integration and than connect that JIRA Integration to a test suite.
+
+A JIRA Authentication consists of: a name, the JIRA Cloud ID, the JIRA url and the JIRA refresh token (obtained after authorizing the app).
+
+A JIRA Integration consists of: a name, the created JIRA authentication id, the JIRA project id and the JIRA issue type id.
+
+!> A test suite can only be linked to a JIRA integration at a time.
```

Comment:
not sure this line makes sense - do you mean to say that only one test suite can be linked at a time?